services/api/Tweek.ApiService.NetCore/Controllers/ContextController.cs

```diff
@@ -30,9 +35,15 @@ public ContextController(IContextDriver contextDriver, JsonSerializer serializer
         {
             if (!_checkAccess(User, new Identity(identityType, identityId))) return Forbid();
             if (data.Count == 0) return Ok();
-
             var identity = new Identity(identityType, identityId);
-            await _contextDriver.AppendContext(identity, data);
+
+            try{
+                await _contextDriver.AppendContext(identity, data);
+            }catch(ArgumentException e)
+            {
+                //log?
```

Comment:
no need, it should be logged by a middleware.

Reply:
How? I am catching the exception, so it will not be an unhandled exception...

Reply:
Now I am using `ILogger`